deployment/docker_python.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# ADAPT THE CONFIGURATION VARIABLES BELOW TO YOUR USE CASE! You will at least
```

Comment:
why can't this script be run in place?

Reply:
What do you mean?

Reply:
Never mind I was just confused about why the DOCKER_PYTHON_SCRIPT needs to be specified

Reply:
I guess one could pass the part that is run inside of Docker as a single string to the docker run command, but I thought that would actually make for a less readable script without any obvious advantages.